composer.json

```diff
@@ -26,7 +26,8 @@
     },
     "require-dev": {
         "mikey179/vfsStream": "~1.5.0",
-        "symfony/var-dumper": "~3.1.3"
+        "symfony/var-dumper": "~3.1.3",
+        "phpunit/phpunit": "^5.4"
```

Comment:
I understand you, but IMHO is too restrictive set a constraint against a specific version. Please, feel free to propose which version you'd like to lock.